presto-main/src/main/java/com/facebook/presto/operator/HashAggregationOperator.java

```diff
@@ -91,6 +92,7 @@ public HashAggregationOperatorFactory(
                     groupByChannels,
                     globalAggregationGroupIds,
                     step,
+                    false,
```

Comment:
Unrelated to your change, but this constructor should have a `VisibleForTesting` annotation.